demos/json-config.html

```diff
@@ -14,5 +14,15 @@
 
 <script src="/bundles/js?modules=o-tracking:/demos/src/demo.js"></script>
 <script src="//registry.origami.ft.com/embedapi?autoload=resize"></script>
+
```

Comment:
Regarding demos, you need to add the markup to a mustache file (if you're going to only build one demo for now, remove the other three), and the configure the config.json file for that demo, and use obt demo to build it